apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -297,6 +308,36 @@ class LivechatClass {
 		});
 		return true;
 	}
+
+	sendRequest(postData: LivechatSendRequestData, attempts = 10) {
+		if (!attempts) {
+			return;
+		}
+		const timeout = settings.get<number>('Livechat_http_timeout');
+		const secretToken = settings.get<string>('Livechat_secret_token');
+		const headers = { 'X-RocketChat-Livechat-Token': secretToken };
+		const options: HTTP.HTTPRequest = {
+			data: postData,
+			...(secretToken !== '' && secretToken !== undefined && { headers }),
+			timeout,
+		};
+		try {
+			const result = HTTP.post(settings.get('Livechat_webhookUrl'), options);
```

Comment:
This `HTTP` is from meteor right? If so, can we avoid using it and use something standard like `axios`?

PS: Sorry I just noticed this so late in review :see_no_evil:

Reply:
changed to axios